### DISCUSSION TITLE
알수없는 에러(Error:-4)로 인해 개발을 전혀 진행할 수 없습니다.

### DISCUSSION THREAD

**Discussion by jake09o** [naverz/zepeto-studio-kor#50] [Issue Report (Legacy)]
![캡처](https://user-images.githubusercontent.com/34004748/145515976-9b50dbda-d667-419b-80f1-01fe140e5836.PNG)
![캡처1](https://user-images.githubusercontent.com/34004748/145515979-efd2ea18-3cbe-44dd-b0ad-73e8ede53fbd.PNG)

프로젝트에 약간의 수정을 가하면 해당 error로 인해 qr코드로 빌드를 할 수 없습니다.

에러 예시)
1. 제페토 탬플릿으로 프로젝트 생성(or 예제 프로젝트 실행)
2. play 및 qr 빌드로 동작 확인(여기가진 제대로 빌드 됩니다)
3. 프로젝트 수정을 위해 GameObject-CreateEmpty(or Cube 및 기타 3D 오브젝트) 생성
4. play 실행(정상 동작 확인)
5. qr 코드로 빌드(시 error:-4와 함께 빌드실패)

에러예시2)
1. 제페토 탬플릿으로 프로젝트 생성(or 예제 프로젝트 실행)
2. play 및 qr 빌드로 동작 확인(여기가진 제대로 빌드 됩니다)
3. 프로젝트 수정을 위해 project view에서 Material을 생성
4. 생성한 material을 Scene에 적용
5. play 실행(정상 동작 확인)
6. qr 코드로 빌드(시 error:-4와 함께 빌드실패)

사용 unity 버전은 2020.3.9f 이며 World 버전은 1.0.4~1.0.6까지 사용하며 확인해봤습니다.
제페토 템플릿을 만드는데 생성한 패키지 파일은 2021.12.01에 다운받았습니다.

설정이 잘못되었나 싶어 다른 컴퓨터에서 설정해도 똑같은 에러로 진행이 되지 않습니다.
해결에 도움을 주시면 감사하겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 여전히 같은 오류가 뜨고 있는지 확인 부탁드리며,
오류가 계속된다면 개발 환경 및 버전을 명시 부탁 드립니다.
감사합니다.
